pkgs/applications/networking/remote/xrdp/default.nix

```diff
@@ -28,29 +41,20 @@ let
 
     preConfigure = "./bootstrap";
 
-    configureFlags = [ "XRDP_CFLAGS=-I${xrdp.src}/common"  ];
+    configureFlags = [ "XRDP_CFLAGS=-I${patchedXrdpSrc}/common"  ];
```

Comment:
Would `srcOnly xrdp` work?  That way it wouldn't be necessary to pull out `patchedXrdpSrc` into a let binding.

Reply:
It results in infinite recursion, sadly. (The `postInstall` of the `srcOnly` derivation still refers to `xorgxrdp`.)